src/test/groovy/org/gradle/android/CrossVersionOutcomeAndRelocationTest.groovy

```diff
@@ -477,8 +477,7 @@ class CrossVersionOutcomeAndRelocationTest extends AbstractTest {
     }
 
     static void android42xOrHigherExpectations(ExpectedOutcomeBuilder builder) {
-        // Should be made cacheable: https://issuetracker.google.com/160138798
-        builder.expect(':app:desugarDebugFileDependencies', SUCCESS)
-        builder.expect(':app:desugarReleaseFileDependencies', SUCCESS)
+        builder.expect(':app:desugarDebugFileDependencies', FROM_CACHE)
+        builder.expect(':app:desugarReleaseFileDependencies', FROM_CACHE)
```

Comment:
Does this task only show up in AGP 4.2?  If so, why do we have the workaround being applied to earlier versions?  Or is it that the task is not triggered to execute in earlier versions?

Reply:
Yeah, good question. This task is not enabled by default in earlier versions but it still exists. Google has decided to enable it by default starting with 4.2.